content/posts/getting-started/netlify/index.fr.md

```diff
@@ -5,17 +5,17 @@ menu:
   sidebar:
     name: Déployer dans Netlify
     identifier: getting-started-netlify
-    parent: getting-started
+    parent: Démarrer
```

Comment:
You can not translate this identifier. This defines the article structure. This probably causing duplicate item on the left sidebar.

Reply:
If I change "Démarrer" to "getting-started", the left menu entry will not be translated into French. exact ?